turbopack/crates/turbo-tasks-backend/src/backend/storage.rs

```diff
@@ -446,8 +446,8 @@ macro_rules! iter_many {
             .iter($crate::data::indicies::$key)
             .filter_map(|(key, value)| match (key, value) {
                 (
-                    &$crate::data::CachedDataItemKey::$key $input,
-                    &$crate::data::CachedDataItemValue::$key { value: $value_pattern }
+                    $crate::data::CachedDataItemKey::$key $input,
+                    $crate::data::CachedDataItemValue::$key { value: $value_pattern }
                 ) $(if $cond)? => Some($iter_item),
                 _ => None,
             })
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +433 to +455]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->I needed to change that as DirtyContainerCount is no longer Copy<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/vercel/next.js/71132?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->